exe/kubernetes-deploy

```diff
@@ -9,7 +9,23 @@ skip_wait = false
 template_dir = nil
 allow_protected_ns = false
 prune = true
+bindings = nil
```

Comment:
You still need to pass this to the `Runner` instance, and I think initializing to `{}` is safer, because if you explicitly pass `bindings: nil` it'll end up trying to merge nil with a hash at runner.rb#L27